packages/bs-flopflip/README.md

```diff
@@ -0,0 +1,26 @@
+# `bs-flopflip`
+
+[BuckleScript](https://github.com/bucklescript/bucklescript) bindings for [flopflip](https://github.com/tdeekens/flopflip) packages - **WIP**
+
+## Install and setup
+
+#### yarn
+
+```bash
+$ yarn add bs-flopflip
```

Comment:
I'd likely prefer having this renamed to `bs-flopflip-react-broadcast` (or whatever scoped package naming convention exists in Bucklescript land. So that we can also add a `bs-flopflip-react-redux` later on. WDYT?

Reply:
It's not really necessary. I would prefer to have one package that contains all the bindings. That's why I named the file `reactBroadcast.re`. There will be one matching each package.

But of course if you prefer to have one for each package we can also do that. We can't however share e.g. the common type definitions.

Reply:
Ah. Disregard my comment below then. I fiddled with having shared TypeScript definitions but resorted to having much of them duplicated as configuration annoyed me a bit. If we can have one BS package, that would be nice. ´@flopflip/bs-flopflip‘ I guess. It would define and depend on both redux and broadcast variations I guess to define types for both. 

I will refactor shared flow- and TypeScript types to be a separate package also if needed.

Reply:
Btw: the "repository index" for reason packages is this one --> https://redex.github.io/

The cool thing is that you can already "submit" a package even if it's not published to npm yet, and it's marked as unpublished to indicate that is e.g. incomplete or experimental.
https://redex.github.io/packages/unpublished